cmd/exit.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	sdkstatus "github.com/nspcc-dev/neofs-sdk-go/client/status"
+	"github.com/spf13/cobra"
+)
+
+// ErrAwaitTimeout represents the expiration of a polling interval
+// while awaiting a certain condition.
+var ErrAwaitTimeout = errors.New("await timeout expired")
+
+// ExitOnErr prints error and exits with a code depending on the error type
+//
+//	0 if nil
+//	1 if [sdkstatus.ErrServerInternal] or untyped
+//	2 if [sdkstatus.ErrObjectAccessDenied]
+//	3 if [ErrAwaitTimeout]
+func ExitOnErr(cmd *cobra.Command, errFmt string, err error) {
+	if err == nil {
+		return
+	}
+
+	if errFmt != "" {
+		err = fmt.Errorf(errFmt, err)
+	}
+
+	const (
+		_ = iota
+		internal
+		aclDenied
+		awaitTimeout
+		alreadyRemoved
+	)
+
+	var code int
+	var accessErr = new(sdkstatus.ObjectAccessDenied)
+	var alreadyRemovedErr = new(sdkstatus.ObjectAlreadyRemoved)
+
+	switch {
+	case errors.Is(err, sdkstatus.ErrServerInternal):
```

Comment:
We won't have these errors here, so this logic (and SDK dependency) doesn't make much sense.